r/adbcdrivermanager/DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: adbcdrivermanager
 Title: 'Arrow' Database Connectivity ('ADBC') Driver Manager
-Version: 0.11.0.9000
+Version: 0.13.0.9000
```

Comment:
This may be `12` not `13`.

Reply:
We never bumped main to 12, so this looks off, but the last release had R at 0.12.0 so this should now be 0.13.0.

https://github.com/apache/arrow-adbc/blob/maint-12/r/adbcdrivermanager/DESCRIPTION#L3

Reply:
It seems that R uses `${LATEST_VERSION}.9000` not `${NEXT_VERSION}.9000` for development version.
https://github.com/apache/arrow/blob/main/r/DESCRIPTION#L3 uses `16.1.0.9000` not `17.0.0.9000`.

@paleolimbot may know the correct value of this.

Reply:
Ah, I see what you mean now. I think you're right.

Reply:
Fixed, thank you both!